backend/src/main/resources/db/changelog/db.changelog-master.yaml

```diff
@@ -4723,4 +4723,105 @@ databaseChangeLog:
             tag: drop-test_order_id-not_null_constraint-from-result
         - dropNotNullConstraint:
             tableName: result
-            columnName: test_order_id
\ No newline at end of file
+            columnName: test_order_id
+  - changeSet:
+      id: create-result-upload-error-table
+      author: nathan@skylight.digital
+      comment: Add a table that holds validation errors for bulk test result uploads
+      changes:
+        - tagDatabase:
+            tag: create-result-upload-error-table
+        - sql:
+            remarks: Create the enumerations for upload error types
+            sql: |
+              CREATE TYPE ${database.defaultSchemaName}.TEST_RESULT_UPLOAD_ERROR as ENUM('HEADER_ERROR', 'INVALID_DATA', 'MISSING_DATA', 'INCORRECT_STRUCTURE');
+        - sql:
+            remarks: Create the enumerations for upload error sources
+            sql: |
+              CREATE TYPE ${database.defaultSchemaName}.RESULT_UPLOAD_ERROR_SOURCE as ENUM('SIMPLEREPORT', 'REPORTSTREAM');
```

Comment:
same as before, change to SIMPLE_REPORT and REPORT_STREAM

Reply:
Done in [4e11732](https://github.com/CDCgov/prime-simplereport/pull/5971/commits/4e11732c96f67f10b616f96abf45822d163bcbff).